app/views/assignments/_grades.html.erb

```diff
@@ -15,39 +15,39 @@
           <tr>
             <td><%= field.name %></td>
             <td><%= grading_field.score %></td>
-            <!-- <td><%= field.points %></td> -->
+            <td></td>
           </tr>
         <% end %>
       <% else %>
         <% @assignment.rubric_fields.order(:id).each do |field| %>
         <% grading_field = @submission.grading_fields.where(rubric_field_id: field.id).first %>
-        <% max_points += field.points.to_i %>
+        <% max_points += field.rubric_field_items.sum(:point) %>
           <tr>
             <td><%= field.name %></td>
```

Comment:
will handle display of item name later. have stuff due tonight and tomorrow night.